mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -292,43 +324,244 @@ def run(self):  # {{{
+                                                        configSectionName,
+                                                        suffix='OHCAnomaly')
+
+            contourLevels = \
+                config.getExpression(configSectionName,
+                                     'contourLevels{}'.format('OHCAnomaly'),
+                                     usenumpyfunc=True)
+
+            plot_vertical_section(config, x, y, z,
+                                  colormap, colorbarLevels, contourLevels,
+                                  colorbarLabel, title, xLabel, yLabel,
+                                  figureName, linewidths=1, xArrayIsTime=True,
+                                  N=movingAveragePointsHovmoller, calendar=calendar)
+
+            filePrefix = self.filePrefixes[2, region]
+            caption = 'Trend of {} OHC Anomaly vs depth from Year 0001'.format(region)
+            write_image_xml(
+                config=config,
+                filePrefix=filePrefix,
+                componentName='Ocean',
+                componentSubdirectory='ocean',
+                galleryGroup='Trends vs Depth',
+                groupLink='trendsvsdepth',
+                thumbnailDescription=u'{} ΔOHC'.format(region),
+                imageDescription=caption,
+                imageCaption=caption)
+
+            if plotOriginalFields:
+                ohc = dsOHC.ohc.isel(nOceanRegionsTmp=regionIndex)
+                ohcScaled = unitsScalefactor*ohc
+                z = ohcScaled.transpose()
+
+                title = 'OHC, {} \n {}'.format(plotTitles[regionIndex], mainRunName)
+
+                figureName = '{}/OHCZ_{}_{}.png'.format(self.plotsDirectory,
+                                                        region,
+                                                        mainRunName)
+            
+                (colormap, colorbarLevels) = setup_colormap(config,
+                                                            configSectionName,
+                                                            suffix='OHC')
+
+                contourLevels = config.getExpression(configSectionName,
+                                                     'contourLevels{}'.format('OHC'),
+                                                     usenumpyfunc=True)
+
+                plot_vertical_section(config, x, y, z,
+                                      colormap, colorbarLevels, contourLevels,
+                                      colorbarLabel, title, xLabel, yLabel,
+                                      figureName, linewidths=1, xArrayIsTime=True,
+                                      N=movingAveragePointsHovmoller, calendar=calendar)
+
+            # Now plot OHC timeseries
 
             # OHC over 0-bottom depth range:
-            ohcTotal = ohc.sum('nVertLevels')
-            ohcTotal = unitsScalefactor*ohcTotal
+            ohcAnomalyTotal = unitsScalefactor*ohcAnomaly.sum('nVertLevels')
```

Comment:
I moved the variables with *no* Anomaly in their names down below, inside the `if plotOriginalFields` statement.